src/python/dxpy/nextflow/nextflow_templates.py

```diff
@@ -0,0 +1,43 @@
+from dxpy.nextflow.nextflow_utils import get_template_dir
+from dxpy.nextflow.nextflow_utils import get_source_file_name
+import json
+import os
+
+
+
+def get_nextflow_dxapp(custom_inputs=[]):
+    """
+    :param custom_inputs: Custom inputs that will be used in created Nextflow pipeline.
+    :type custom_inputs: list
+
+    Creates Nextflow dxapp.json from the Nextflow dxapp.json template
+    """
+    with open(os.path.join(str(get_template_dir()), 'dxapp.json'), 'r') as f:
+        dxapp = json.load(f)
+    dxapp["inputSpec"] = custom_inputs + dxapp["inputSpec"]
+    return dxapp
+
+
+def get_nextflow_src(inputs=[], profile=None):
```

Comment:
Regarding passing the default empty list please check https://stackoverflow.com/questions/366422/what-is-the-pythonic-way-to-avoid-default-parameters-that-are-empty-lists